src/domain/core/src/entities/metadata_chain.rs

```diff
@@ -481,22 +481,29 @@ pub enum AppendValidationError {
     #[error(transparent)]
     InvalidEvent(#[from] InvalidEventError),
     #[error(transparent)]
-    NoOpEvent(#[from] NoOpEventError),
+    NoOpEvent(#[from] Box<NoOpEventError>),
```

Comment:
Maybe Box for the event structure inside the error?

Reply:
Done https://github.com/kamu-data/kamu-cli/pull/473/commits/122e88a7b4092a9507fea49cda1f10a747119054